wadas/ui/configure_ftp_cameras_dialog.py

```diff
@@ -220,6 +224,10 @@ def accept_and_close(self):
                         # Add FTP Camera to FTP server users list
                         self.add_camera_to_ftp_server(cur_ui_id, camera_ftp_path, camera_pass)
 
+                        # Insert camera into db if enabled
+                        if (db:=DataBase.get_instance()) and db.enabled:
```

Comment:
Check and cache db on the local variable, at the beginning of the method, and then use it over the code (below, in other code snippets, you're repeating the same statements).